feature-configs/base/performance/operator_subscription.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: performance-addon-operator-subscription
   namespace: openshift-performance-addon
 spec:
-  channel: alpha
+  channel: 0.0.1
```

Comment:
I think this is unrelated to this PR

Reply:
Yup, it was to make CI happy upfront. Will wait for the channel fix pr to land first.